src/js/components/TextArea/doc.js

```diff
@@ -35,3 +35,35 @@ Only use this when the containing context provides sufficient affordance.`,
 
   return DocumentedTextArea;
 };
+
+export const themeDoc = {
+  'global.colors.placeholder': {
+    description: 'The placeholder color used for TextArea.',
+    type: 'string',
+    defaultValue: '#AAAAAA',
+  },
+  'global.control.border.width': {
+    description: 'The border width.',
+    type: 'string',
+    defaultValue: '1px',
+  },
+  'global.input.weight': {
+    description: 'The font weight of the label.',
+    type: 'number',
+    defaultValue: 600,
+  },
+  'global.focus.border.color': {
+    description: `The color of the border when component is focused.`,
+    type: 'string | { dark: string, light: string }',
+    defaultValue: 'focus',
+  },
+  'global.spacing': {
+    description: 'The padding of the text.',
```

Comment:
I think we should be slightly more detailed here. `global.spacing` isn't the final padding used. It's `global.spacing / 2`.

Reply:
@ericsoderberghp Yes we can. since the value varies I wan't sure if it is OK to put a varaible on the defalut value.
Currently, the value is `spacing: '${baseSpacing}px',` what will be a reasonable and readble way to present it on the default value?

Reply:
I'm wondering if we should put the math in the theme and have `theme.global.input.padding` that is set to `baseSpacing / 2`.